requirements.dev.txt

```diff
@@ -15,7 +15,6 @@ pytest-markdown-docs==0.4.3
 pytest-timeout~=2.1.0
 python-dotenv~=1.0.0;python_version>='3.8'
 ruff~=0.0.239
-synchronicity==0.4.1
```

Comment:
Removing synchronicity from the dev dependencies since it's a dependency of the package itself and the double specification seems error-prone